Collect feedback deactivation

truongwp opened the pull request:
Handoff: https://app.clickup.com/9006050445/v/dc/8ccuv4d-30651

Feedback form: https://feedback.strategy11.com/wp-admin/admin.php?page=formidable&frm_action=edit&id=20

truongwp commented:
Thanks @garretlaxton. I will fix the icon error. 
I discussed with Dalton and decided to show this popup once every 6 months. Maybe I will update that time to 1 hour or less so you can test it more easily. I will tell you when it's done.

truongwp commented:
@garretlaxton I fixed the icon issue.

Now the form will show once in 15 minutes. Please test it again.

garretlaxton reviewed (approved):
Thanks, this works great now!

Crabcyborg reviewed (approved):
This all looks great!

I haven't tested this yet but I will soon and merge this for next release. I plan to wait a day or two in case something gets reported in version 6.14 that needs a patch.

I left one comment about moving the CSS into a file. That's optional. I think we're fine merging this as-is.

🚀

truongwp commented on classes/controllers/FrmDeactivationFeedbackController.php:
```diff
@@ -0,0 +1,221 @@
+	private static function modal_style() {
+		?>
+		<style>
+			#frm-deactivation-modal { --padding: 24px; }
```
I moved this into a CSS file.

truongwp commented:
@Crabcyborg Do not merge this before we update the expired time of the popup. Laura is also testing this.

lauramekaj1 commented:
Hi @truongwp, I have done some testing on the deactivation feedback popup and here are some findings if you can review them!

1. If I expand the input text field, the whole modal expands and making it impossible to click on the buttons. Maybe it would be good to have a scroll also on the modal itself! (https://imgur.com/PUEQBgh).
1.1 Also, if I select other options and then go back to the option where I expanded the field, the text will be erased but the field will remain expanded as it previously was. Should it also go back to the initial size? (https://imgur.com/VyOoujj)
2. If I click Submit&Deactivate, the red border on the input text field just keeps showing whenever I click the button and does not remain red until I do an outside click on the modal. Here is the screen recording:(https://www.loom.com/share/6048c3b1d2b246cfa33b4dbb92ff146f?sid=6d694272-11a1-491a-aad2-b4c6e2859a23). 
Maybe if you also agree, I think it would be better to show a message like "Please provide a reason!" just to let the users know that those are required fields which need to be filled!

truongwp commented:
Thanks @lauramekaj1. I fixed those problems in the feedback form. You won't need to update the code to test again.

For the 1st: `Should it also go back to the initial size?`: This is hard, I think we will keep it as is.
For the 2nd, users should be able to submit the form with those inputs blank. Only if no radio option is selected, it will show an error message. I updated the message.

Crabcyborg reviewed (approved):
I'll test this before I release it tomorrow.

But I'll go ahead and merge this now. It all looks good to me.

Thanks @truongwp, @garretlaxton, and @lauramekaj1 for working on this!

🙌
